test/helper.lua

```diff
@@ -0,0 +1,43 @@
+require('strict').on()
+
+local log = require('log')
+local digest = require('digest')
+local fio = require('fio')
+
+local ok, cartridge_helpers = pcall(require, 'cartridge.test-helpers')
+if not ok then
+    log.error('Please, install cartridge rock to run tests')
+    os.exit(1)
```

Comment:
Looks a bit strange. I suggest to skip in case if cartridge is not found.

Reply:
Thanks for noticing. Fixed in fff3a7e1132b6e28b965ba383494b2d9664a4a59.